server/apps/distributed-app/docs/modules/ROOT/pages/configure/queue.adoc

```diff
@@ -0,0 +1,14 @@
+= Distributed James Server &mdash; queue.properties
+:navtitle: queue.properties
+
+This configuration helps you configure mail queue you want to select.
+
+== RabbitMQ Configuration
+
+.rabbitmq.properties content
+|===
+| Property name | explanation
+
+| queue.choice
+| Mail queue can be implemented by many technologies, Pulsar, RabbitMQ,.... This property will choose which mail queue you want
```

Comment:
Name exactly which mail queue options are available (RABBITMQ / PULSAR), and which is the default choice.